### PR TITLE
Fix picker

### PR DESCRIPTION
**Pull request by laurennlam**
The user matrix need to be transpose before being used to project points

### CODE REVIEW

**Comment by jourdain** [comment]
@martinken ?
@laurennlam Ready to merge?

**Comment by laurennlam** [comment]
Yes, Ready to merge !
It's a fix if the user transform of an actor has been changed.

File: Sources/Rendering/Core/Picker/index.js [hunk 1]
```diff
@@ -209,8 +214,8 @@ function vtkPicker(publicAPI, model) {
         scale[1] = vtkMath.norm(col2);
         scale[2] = vtkMath.norm(col3);
 
-        vec3.transformMat4(p1Mapper, p1World, model.transformMatrix);
-        vec3.transformMat4(p2Mapper, p2World, model.transformMatrix);
+        vec4.transformMat4(p1Mapper, p1World, model.transformMatrix);
+        vec4.transformMat4(p2Mapper, p2World, model.transformMatrix);
```
Comment (martinken) [inline Sources/Rendering/Core/Picker/index.js]: Do you need divide p*Mapper by p*Mapper[3] here? I think you should to be safe.

Comment (martinken) [reply Sources/Rendering/Core/Picker/index.js]: Ping

Comment (laurennlam) [reply Sources/Rendering/Core/Picker/index.js]: Yes, sorry I just made the correction after your response. But yes, to be more efficient and understandable, it should be safe do divide pMapper by pMapper[3].

**Comment by laurennlam** [comment]
Is it okay now ?

**Pull request merged**